impeller/display_list/canvas.cc

```diff
@@ -1667,10 +1667,15 @@ bool Canvas::BlitToOnscreen() {
   auto offscreen_target = render_passes_.back()
                               .inline_pass_context->GetPassTarget()
                               .GetRenderTarget();
-
+  // Unlike other backends the blit to restore the onscreen fails for GLES
+  // if the src is a multisample framebuffer, even if we specifically target
+  // the non-multisampled resolve of the dst.
+  bool is_gles_and_must_skip_blit = renderer_.GetContext()->GetBackendType() ==
```

Comment:
It seems like once we have a multisampled renderbuffer attached to a texture blits will just fail :(